src/api.js

```diff
@@ -156,7 +156,7 @@ API.prototype.getTicker = function (){
 API.prototype.getUnspent = function (fromAddresses, confirmations){
   var data = {
       active : fromAddresses.join('|')
-    , confirmations : Helpers.isPositiveNumber(confirmations) ? confirmations : 0
+    , confirmations : Helpers.isPositiveNumber(confirmations) ? confirmations : -1
```

Comment:
-1 will get all unspent outputs regardless of priority / confirmations, null will cause a server error: "Invalid Numerical Value".